ext/cfx-ui/src/app/servers/components/list/servers-list-item.component.scss

```diff
@@ -92,6 +92,7 @@
 		bottom: 2px;
 
 		width: var(--q4);
+		pointer-events: none;
```

Comment:
The 'premium' gradient was positioned over the badge for some reason - by dropping pointer events for it, the tooltip at least does work.